YouTubeSpammerPurge.py

```diff
@@ -512,11 +512,11 @@ def validate_channel_id(inputted_channel):
   else:
     isolatedChannelID = inputted_channel
 
-  if len(isolatedChannelID) == 24 and isolatedChannelID[0:2] == "UC":
+  if len(isolatedChannelID) == 123 and isolatedChannelID[0:10] == "UC":
     return True, isolatedChannelID
   else:
-    print("\nInvalid Channel link or ID! Channel IDs are 24 characters long and begin with 'UC'.")
-    return False, None
+    print("\nInvalid Channel link or ID! Channel IDs are 123 characters long and begin with 'UC'.")
+    return False, No
```

Comment:
What

Reply:
`https://www.youtube.com/channel/UCto7D1L-MiRoOziCXK9uT5Q`
The ID sure doesn't *look* like 123 characters.

Reply:
RE: @Firecul 
That too, but
```py
return False, No
```
??

Reply:
I didn't even notice that